### DISCUSSION TITLE
Servo motor Arduino

### DISCUSSION THREAD

**Discussion by SANTILIZER** [bitcraze/discussions#781] [Q&A]
Hello every one I am new to Arduino and I am facing the following problem,
I have a problem with one of my servo motor (mg996R) it has the following Problems :
1- It is very shaky when i fix it to a certain angle for example if i tell it to go to angle 110 it will go there but it cant stand still 
2- After a certain time around 10 to 20 second (not always fixed time it can be below or above)  it  neglects all orders and starts moving on its own trying to break it is limited movement . meaning that if the motor is set on 110 angle suddenly it will start spinning to reach 0 angle and keeps the spinning  although it cant go beyond that ( even manually ) . Uploading a new Arduino code will not fix it nor cutting the power supply for the servos (it will start spinning once the power is back), it can only be fixed by cutting the supply for the servos then upload a new code to the Arduino.
I wish to know what is the cause if this issue because I cant find a solution to it any where ( and I did glue it to my intended part so simply changing it is quiet hard)

**Comment by krichardsson** [comment]
Sorry, this forum is not for Arduino nor servo motors. Closing as off topic.
